Lib/Environment.cpp

```diff
@@ -119,51 +114,6 @@ int Environment::remainingTime() const
   return options->timeLimitInDeciseconds()*100 - timer->elapsedMilliseconds();
 }
 
-/**
- * Acquire an output stream
- *
- * A process cannot hold an output stream during forking.
- */
-void Environment::beginOutput()
-{
-  ASS_GE(_outputDepth,0);
-
-  _outputDepth++;
-}
-
-/**
- * Release the output stream
- */
-void Environment::endOutput()
-{
-  ASS_G(_outputDepth,0);
-
-  _outputDepth--;
-  if(_outputDepth==0) {
-    cout.flush();
```

Comment:
Shell we theoretically be missing some flushings now?

Reply:
Theoretically. In practice most call sites use `std::endl` somewhere which will cause an equivalent flush. Also not sure the OS is required to obey a runtime flush request, so it may never have done what we wanted.